src/mudlet.cpp

```diff
@@ -4405,6 +4952,14 @@ QString mudlet::getMudletPath(const mudletPathType mode, const QString& extra1,
         // directory for that profile - does NOT end in a '/' unless the
         // supplied profle name does:
         return QStringLiteral("%1/.config/mudlet/profiles/%2").arg(QDir::homePath(), extra1);
+    case profileSoundsPath:
+        // Takes one extra argument (profile name) that returns the directory
+        // for the profile game save sounds files - does NOT end in a '/'
+        return QStringLiteral("%1/.config/mudlet/profiles/%2/sound").arg(QDir::homePath(), extra1);
+    case profileSoundPathFileName:
+        // Takes two extra arguments (profile name, mapFileName) that returns
```

Comment:
And this: `// Takes two extra arguments (profile name, soundFileName) that returns`

Reply:
Resolved.